case.jsonld

```diff
@@ -1583,6 +1583,102 @@
             "uco-core:target": {
                 "@id": "kb:dce632ef-83f0-5481-9b10-d38455d155be"
             }
+        },
+        {
+            "@id": "kb:b5952e3b-5e8a-5059-a2fa-fb1e3d727226",
+            "@type": "uco-observable:ObservableObject",
+            "uco-core:hasFacet": [
+                {
+                    "@id": "kb:1ea0f9f9-a53b-50c6-be1c-b4fd46b3569b",
+                    "@type": "uco-observable:OperatingSystemFacet",
+                    "uco-observable:environmentVariables": {
+                        "@id": "kb:4e32a8c7-9d4b-5f8d-aa81-6ab6905814fe",
+                        "@type": "uco-types:Dictionary",
```

Comment:
I didn't see this class in `types.py`. Since the object's being generated now, it should be added to the API.

Reply:
Implemented in 6cb5844.